specification/paths/Returns-v1-shops-shop_id-reasons-return_reason_id-questions.json

```
@@ -0,0 +1,83 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-shop_id"
+    },
+    {
+      "$ref": "#/components/parameters/path-return_reason_id"
+    }
+  ],
+  "post": {
+    "tags": [
+      "ReturnQuestions"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
+        ]
+      }
+    ],
+    "summary": "Link return questions to the given return reason.",
+    "description": "This endpoint links return questions to the specified return reason.",
+    "requestBody": {
+      "description": "The return question ID's to link to the return reason",
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "type": "object",
+                "required": [
+                  "return_question_ids"
```

```suggestion
                  "return_questions"
```

I think this should be called questions since we aren't supplying id's